migration/src/androidTest/kotlin/kin/sdk/migration/MigrationManagerIntegrationTest.kt

```diff
@@ -26,19 +26,21 @@ import kotlin.test.fail
 @Suppress("FunctionName")
 class MigrationManagerIntegrationTest {
 
-    private val sdkTestNetworkUrl = "http://horizon-testnet.kininfrastructure.com/"
+    private val sdkTestNetworkUrl = "https://horizon-testnet.kininfrastructure.com/"
     private val sdkTestNetworkId = "Kin Testnet ; December 2018"
     private val coreTestNetworkUrl = "https://horizon-playground.kininfrastructure.com/"
     private val coreTestNetworkId = "Kin Playground Network ; June 2018"
     private val coreIssuer = "GBC3SG6NGTSZ2OMH3FFGB7UVRQWILW367U4GSOOF4TFSZONV42UJXUH7"
+    private val migrateServiceUrl = "https://migration-devplatform-production.developers.kinecosystem.com/migrate?address="
```

Comment:
why production url?

Reply:
mistake but anyway all the integration tests are been developed in a different branch and in that branch everything is up to date